src/ILCompiler.Compiler/src/CppCodeGen/NodeDataSection.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ILCompiler.Compiler.CppCodeGen
+{
+    public enum NodeDataSectionType
+    {
+        Relocation,
+        ByteData
+    }
+
+    public struct NodeDataSection
```

Comment:
The style that we have standardized on for struct like this is readonly fields and constructor to initialize them (https://github.com/dotnet/corert/issues/249). Could you please update this one to match?